.github/workflows/trigger_azdo_ci.yml

```diff
@@ -14,31 +17,31 @@ jobs:
     steps:
     - name: Trigger Main CI
       uses: Azure/pipelines@v1
-      if: ${{ github.repository == 'ni/grpc-device' && github.ref == 'refs/heads/main' }}
+      if: ${{ github.repository == 'ni/grpc-device' && github.event.workflow_run.head_branch == 'main' }}
```

Comment:
Updated these because the github.ref for workflow_run triggers will always be the repo's default branch (or main). Using the `github.eventworkflow_run.head_branch` instead will make sure we're triggering the right AzDo pipeline based on where the CI completed (main or a releases branch).